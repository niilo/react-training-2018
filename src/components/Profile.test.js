import React from 'react';
import renderer from 'react-test-renderer';
import Profile from './Profile';

it('renders default correctly', () => {
  const props = {
    info: {
      name: 'react',
      location: 'location',
      company: 'Facebook',
      followers: 1,
      following: 1,
      public_repos: 2,
      html_url: 'https://github.com/facebook/react',
      stargazers_count: 73600,
      login: 'facebook',
      blog: 'https://github.com/facebook/react',
      avatar_url: 'https://avatars3.githubusercontent.com/u/69631?v=4',
    },
  };

  const tree = renderer.create(<Profile {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});
