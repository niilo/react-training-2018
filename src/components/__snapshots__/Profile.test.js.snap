// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders default correctly 1`] = `
<div>
  <div
    className="column"
  >
    <img
      alt="Avatar for facebook"
      className="avatar"
      src="https://avatars3.githubusercontent.com/u/69631?v=4"
    />
    <h2
      className="username"
    >
      @
      facebook
    </h2>
  </div>
  <button
    className="reset"
    onClick={[Function]}
  >
    Reset
  </button>
</div>
`;
